homeassistant/components/tplink/light.py

```diff
@@ -64,12 +64,15 @@ async def async_setup_entry(hass: HomeAssistantType, config_entry, async_add_ent
     devices = hass.data[TPLINK_DOMAIN][CONF_LIGHT]
     entities = []
 
+    await hass.async_add_executor_job(get_devices_sysinfo,devices)
     for device in devices:
-        await hass.async_add_executor_job(device.get_sysinfo)
         entities.append(TPLinkSmartBulb(device))
 
     async_add_entities(entities, update_before_add=True)
 
+def get_devices_sysinfo(devices):
+    for device in devices:
+        device.get_sysinfo
```

Comment:
You do not call the method, that's why the linter says that there is no effect.

```suggestion
        device.get_sysinfo()
```

Reply:
Yeah, been rushing a bit this morning :(